client/bundle.js

```diff
@@ -0,0 +1,280 @@
+(function(){function r(e,n,t){function o(i,f){if(!n[i]){if(!e[i]){var c="function"==typeof require&&require;if(!f&&c)return c(i,!0);if(u)return u(i,!0);var a=new Error("Cannot find module '"+i+"'");throw a.code="MODULE_NOT_FOUND",a}var p=n[i]={exports:{}};e[i][0].call(p.exports,function(r){var n=e[i][1][r];return o(n||r)},p,p.exports,r,e,n,t)}return n[i].exports}for(var u="function"==typeof require&&require,i=0;i<t.length;i++)o(t[i]);return o}return r})()({1:[function(require,module,exports){
+// The console.log is because we did not manage to return anything as of yet
+const recipes = require("../server/data/main");
+
+const getMainRecipes = async () => {
+    const response = await fetch("http://localhost:8080/api/recipes");
+    const data = await response.json();
+    console.log(data);
+    return data;
+};
+
+const getRandomRecipe = async () => {
+    const response = await fetch("http://localhost:8080/api/recipes/random");
+    const data = await response.json();
+    console.log(data);
+    return data;
+};
+
+const searchRecipes = async (query) => {
+    const response = await fetch(
+        `http://localhost:8080/api/recipes/search?q=${query}`
+    );
+    const data = await response.json();
+    console.log(data);
+    return data;
+};
+
+const resultsBtn = document.querySelector("#resultsBtn");
+const randomBtn = document.querySelector("#randomBtn");
+const searchBar = document.querySelector("#searchBar");
+
+resultsBtn.addEventListener("click", resultRecipes);
```

Comment:
32: resultRecipes is not defined - the addEventListener calls the function. Would you want to call the searchRecipes function here?